Contact/Search.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Nate.ContactApp
+{
+    public static class Search
+    {
+        private static string lastSearchString = string.Empty;
+        private static List<Contact> lastSearch;
+
+        public static List<Contact> Filter(List<Contact> ContactList, string parameter)
```

Comment:
I wouldn't make this a static method.  If anything make a SearchRepository that implements and interface that you can later mock.  Otherwise it will be hard to test this when you actually start using a database.

Reply:
This is admittedly a very rudimentary filter function and needs a lot of work. At this point I just separated it from the actual repository classes and after figuring a way to get it to compile while saving a previous search in order to more efficiently filter results I left it alone as this was not something I knew was needed in the short term.

Reply:
I will be digging into this more tomorrow.